server/db/models/product.js

```diff
@@ -31,6 +31,15 @@ const Product = db.define('product', {
       notEmpty: true
     }
   },
+  // gallery: {
+  //   type: STRING,
+  //   get: function() {
+  //     return JSON.parse(this.getDataValue('gallery'))
+  //   },
+  //   set: function(val) {
+  //     return this.setDataValue('gallery', JSON.stringify(val))
+  //   }
+  // },
```

Comment:
Following the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle, if we aren't using this code in this PR, let's remove it from here, and bring it back later if you decide you do want to use it.

Reply:
yes.